README.md

```diff
@@ -96,7 +136,7 @@ Evaluate `SenseNova-SI-1.1-InternVL3-8B` on `MindCubeBench_tiny_raw_qa`:
 ```bash
 python run.py --data MindCubeBench_tiny_raw_qa \
               --model SenseNova-SI-1.1-InternVL3-8B \
-              --verbose --reuse
+              --verbose --reuse --judge extract_matching
```

Comment:
What about adding one more example when `--judge gpt-4o-1120`? It may include setting some `ENV` before evaluating?